### ISSUE TITLE
A bug about testing the acc of val and train datasets after per epoch 

### ISSUE THREAD

**Issue by weiyuuuuyiew** [YCAyca/3D-Multi-Object-Tracking-using-Lidar#7]
Thank you very much for your contribution to the training set and validation set testing after each epoch. But I found a bug when I run it, the val_acc can be calculate correctly, but the train_acc is wrong.
![1](https://github.com/user-attachments/assets/6fa8251e-b14f-4f8a-a04f-f15decb05c84)
![2](https://github.com/user-attachments/assets/1e4b5690-3a53-44b1-8d0e-5eb600a30bd6)

**Comment by weiyuuuuyiew** [comment]
the first is val_acc, second is train_acc, the parameters are same except the dataloader, maybe the issue with dataloader? and I found the recall_rcnn are both norm....sorry, my english is too bad ,cant express clearly

**Comment by Petros626** [comment]
I think this repo isn't maintained anymore? @YCAyca
